### PR TITLE
Github Pages deploy

### PR DESCRIPTION
**Pull request by mattborghi**
This PR adds the functionality to deploy the repository to github pages.

Current steps and remaining tasks are:

- [x] Deploying to Github Pages using a script via `package.json`
- [ ] Adding a Github Action that runs the above script on merged PRs or commits to the `master` branch.
- [ ] Remove verbosity from console.
- [ ] Add custom domain. More information [here](https://github.com/tschaub/gh-pages#deploying-to-github-pages-with-custom-domain), although this can be changed from the Settings panel from the repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
   "description": "",
+  "homepage": "https://ekumenlabs.github.io/telegram-tt",
   "main": "index.js",
   "scripts": {
     "dev": "cross-env APP_ENV=development webpack serve --env mode=dev --env isDevServer --mode development --config ./webpack.config.js",
     "build": "webpack --mode production",
     "build:staging": "rm -rf dist/ && APP_ENV=staging npm run build && ./deploy/copy_to_dist.sh",
     "build:production": "npm i && rm -rf dist/ && APP_VERSION=$(npm run inc_version --silent) APP_ENV=production npm run build && ./deploy/copy_to_dist.sh",
     "deploy:production": "npm run build:production && git add -A && git commit -a -m '[Build]' --no-verify && git push",
+    "deploy:staging": "npm run build:staging && gh-pages -d dist --message '[EKUDEPLOY] Deployed to Github Pages'",
```
Comment (german-e-mas) [inline package.json:13]: Let's have this for `production`.

File: src/components/middle/immedia/constants.ts [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import { DEBUG } from '../../../config';
+// import { DEBUG } from '../../../config';
 
-const WEBSOCKET_URL = DEBUG
-  ? 'http://localhost:3000/ws'
-  : 'http://immedia.herokuapp.com/ws';
-// const WEBSOCKET_URL = "http://immedia.herokuapp.com/ws";
+// uncomment for production
```
Comment (german-e-mas) [inline src/components/middle/immedia/constants.ts:3]: I believe we can set `APP_ENV` env var to `production` to avoid the uncomment.

**Review by german-e-mas — COMMENTED** [review]
Thanks @mattborghi ! Let's have this for deploying to `production`. The gh-page address will be the same, and I can change the reference from the DNS side.

File: package.json (hunk 1 above)
Comment (mattborghi) [reply package.json]: Done in 6f4ef31293c0de939c0fbe4c10ff5992ad42a366

File: src/components/middle/immedia/constants.ts (hunk 2 above)
Comment (mattborghi) [reply src/components/middle/immedia/constants.ts]: Done in 2f6cb40de8bb82305d24d12e807ecd047bb165c5

**Review by german-e-mas — APPROVED** [review]
LGTM!

**Pull request merged**